helm/bridge-operator-chart/templates/deployment.yaml

```diff
@@ -51,21 +51,21 @@ spec:
           mountPath: /etc/ssl/certs/
         ports:
         - name: http
-          containerPort: 8000
+          containerPort: 18000
```

Comment:
This is probably not needed since it listen on host network, but let's havy it in sync